app/src/main/res/values/strings.xml

```diff
@@ -18,49 +18,49 @@
+    <string name="crash_title">EteSync crashed!</string>
     <string name="crash_message">Please send stack trace to developers.</string>
     <string name="crash_email_subject">EteSync Debug Info</string>
-    <string name="crash_email_body">If possible, please include any other relevant information such as what you did to trigger this error.</string>
+    <string name="crash_email_body">If possible, please include any other relevant info, such as what you did before this happened.</string>
 
     <!-- tourguide -->
     <string name="tourguide_title">Did you know?</string>
 
     <!-- startup dialogs -->
     <string name="startup_battery_optimization">Battery Optimization</string>
-    <string name="startup_battery_optimization_message">Android may disable/reduce EteSync synchronization after a few days. To prevent this, turn off battery optimization.</string>
+    <string name="startup_battery_optimization_message">Turn off battery optimization to prevent Android turning off/reducing EteSync syncing after a few days.</string>
     <string name="startup_battery_optimization_disable">Turn off for EteSync</string>
     <string name="startup_dont_show_again">Don\'t show again</string>
     <string name="startup_development_version">EteSync Preview Release</string>
-    <string name="startup_development_version_message">This is a development version of EteSync. Be aware that things may not work as expected. Please give us constructive feedback to improve EteSync.</string>
-    <string name="startup_development_version_give_feedback">Give feedback</string>
+    <string name="startup_development_version_message">This is a development version of EteSync. Be aware that things may not work as expected. Please give us constructive feedback to improve it.</string>
+    <string name="startup_development_version_give_feedback">Provide feedback</string>
     <string name="startup_vendor_specific_bugs">Potential Vendor Bugs</string>
-    <string name="startup_vendor_specific_bugs_message">EteSync has detected you are using an Android version that may contain vendor specific bugs.\nPlease take a look at the FAQ for more information.</string>
+    <string name="startup_vendor_specific_bugs_message">You are using an Android version that may contain vendor specific bugs.\nPlease take a look at the FAQ for more info.</string>
     <string name="startup_vendor_specific_bugs_open_faq">Open FAQ</string>
 
     <!-- AboutActivity -->
-    <string name="about_license_terms">License terms</string>
-    <string name="about_license_info_no_warranty">This program comes with ABSOLUTELY NO WARRANTY. It is free software, and you are welcome to redistribute it under certain conditions.</string>
+    <string name="about_license_terms">License info</string>
+    <string name="about_license_info_no_warranty">This copylefted libre software licensed GPLv3+ comes with ABSOLUTELY NO WARRANTY. Use, see, change and share at will, with all.</string>
```

Comment:
This change is a bit problematic as it removes the "under certain conditions" part.

Reply:
I haven't seen "you are welcome to redistribute it under certain conditions" anywhere else that I can think of. The GPLv3+ doesn't allow any invariant sections, so "Use, see, change and share at will, with all." isn't a functional part of the license.

The original string lacks the license altogether, and "free software" without a qualifier doesn't communicate much of anything.

Reply:
Look at the app, this string is immediately followed with the license in its entirety. I agree the existing string is lacking though.
How about this (sort of taken from the FSFs website):
This program is free software: you can redistribute it and/or modify it under the terms of the license below.

Reply:
The FSF has been at it for 35 years trying to communicate to communicate that point, with no real distinction made.
Then it is all fine as it is. Near nobody reads the whole license text to find out what a sentence means. It isn't like people just happen to get ideas about GPL-licensed software, so to do anything functional with it, those people already know.
To separate it from other software, in clarifying how it is functionally different to freeware or closed source software is the issue IMO.

Reply:
My problem with the suggested translation is twofold:
1. "copyleft" and "libre" are words you and I understand, because we know what the GPL is, but for a layman, it won't add any information. So I don't think they add clarity.
2. The second part implies that it comes with no restrictions, which is not true, per the GPL. I'm also not sure about the wording.

Maybe add a link to a website that shows a summary of what's allowed and what isn't (there are a few)? Even github has a summary: https://github.com/etesync/android/blob/master/LICENSE (though let's not link to github).

Reply:
1. copylefted qualifies libre, as something in addition to. In turn these are qualities of the software, as opposed to "free software" being a concept to people already in the know. Besides, copyleft is stricter than free software, as free software could also be permissive. With the shortest possible explanation of what that entails, I don't know what could be added to make things clearer.
2. I added a semicolon to point out that "with all" is actually a "restriction", requirement, or protection if you will.
"under certain conditions" has the quality of being both vague and not sounding nice. It says "here you go, there is the whole license to figure out what that means". Like someone was forced to admit to it.

It isn't untrue now, and it is "short term"→"short description"→"full license". Makes sense to me. A link would be great.
Is https://choosealicense.com/licenses/gpl-3.0/ the one made by GitHub? (Google Analytics)
https://simple.wikipedia.org/wiki/GNU_General_Public_License is OK, and doesn't load any cancer.
Wikipedia makes it very easy for translators to find something in their language to link to.

Reply:
Yeah, I didn't mean we would link to Github. The simple wikipedia looks OK, but there isn't one for "simple French", or for any other language, so we are not solving anything really.
To be honest, the more we discuss it, the more I think the about place is not the right place to educate users about FLOSS and licensing. The license is there, if they are interested they can search more online. I'm even fine with just removing this string altogether. It doesn't add value and it just makes things complex.

Reply:
Using the simple one means it as simple as possible for everyone, and there are variants in local languages for others.
The place is exactly the place. Educate users, get donations, all that stuff. I can help with that.

Reply:
OK, I'm OK with the wikipedia link. What have we settled on then?
How about: "EteSync is free and open-source software. You can use, modify and distribute it freely under the terms of the license. For more information: [LINK]"

Reply:
@tasn If by that you mean "gratis", then I suppose it is already known to the user at that point. If both, then it is highly confusing that one doesn't qualify the other, which is my problem with "FOSS" as a term. It also isn't "open-source", but "Open Source", and that doesn't have trademark protection in other languages, where it is just a confusing term.
"Free" sells it short, and so does "open-source".

Reply:
I'm very familiar with software licensing. I know what gratis and libre mean. I know about OSS, and FOSS, and FLOSS. It is in fact open-source. There is also a trademark called Open Source, but open-source is a correct term and common.
I told you, I don't think the about page is a proper place for educating users about FLOSS, and the difference between gratis and libre. I like how it was in the beginning and I think every version suggested since then was worse, not better.
I'm against changing this at all.

Reply:
Here are my issues with it.
1. It says "program" when "app" is used elsewhere. Sometimes "EteSync" self-references, so it is arguably a third way to do one thing.
2. Capital letter words shown to the user is bad practice. Looks hostile.
3. The idea that it doesn't come with a warranty of any kind isn't important to point out, and it just looks unprofessional. It is in the license. It is the least valuable thing about the license.
4. It also comes first for no good reason, so that is the third way it undermines value.
5. "free software" is meaningless now that people don't use "freeware", and it _is_ the way that people describe "gratis software". The user will likely conclude it is ~freeware.
6. "free software" is likely to be mistranslated into "gratis software", which just adds insult to injury.
7. "Free software" is not specifically "copyleft", which sells the project short. Similarly EteSync is also "software", but not everyone can say they are copyleft. That is an extra restriction/requirement/protection.
8. The Oxford comma ", and" makes it seem like redistribution is different from free software, so there is even less chance of conveying what it "free software" means.
9. "welcome to" is nice, but "under certain conditions" is not. Looks shady. The user is none the wiser. Relegating the user to the full license to figure it out is bad, because the license text is rough. Near nobody will read the full license, especially if they are confused at this point.
10. The whole license is dumped on the user after that. Never seen that before, and to me it looks a lot like a wall of text.
11. I don't know what is at the bottom of the license text, and I have to scroll through it. Fairly small problem, but it doesn't exactly fill me with confidence (for something I am unlikely to be very good at).
12. It all doesn't fit on one screen, which is unaesthetic. The category is up top too, so it is also unfamiliar. Usually it is at the bottom.

In about the same amount of initial text, (in logical succession) the user can both be introduced to functional words that _expand_ on what is merely free software/Open Source/etc. is (because EteSync is copyleft). It also works in all languages, all while explaining the concept of copyleft software in simple terms. I know that works, because I have done it in many projects. My experience is it increases donations. See https://liberapay.com/explore/teams

Reply:
Just tried it on an unsuspecting user:
Expectation about the category is that it is going to be about the license, and the people that made the program.

User scrolls down the side without reading, and notices there is a lot of text. Is somewhere into the GPLv3 text, and isn't going to read it. We are failing there.
When asked about the top part of the text, the user thinks there is no warranty because the program was gratis. (so failing to understand this is because of the license). 
Says "redistribute" means "this is where you have to read the full text", again isn't going to do so. That is the second fail where the current EteSync shouldn't fail at presenting its strongest attribute.

User knows about open source (calls it open code though), thinks "free software" used here might mean one or the other, or both, asks me which one it is.
I say that it could be _free software_ and still cost money. User derives that "the unique thing is that the source is free". I answer yes.

Reply:
As per my other comment, I just tried it on an unsuspecting translator, and it's indeed a pain to translate. Legalese are a problem like that. "copyleft" won't be easy to translate either. And heck, I don't think this page is even used often nor do I care about it that much.
How about:
This program is free (libre) and open-source software. You can use, modify and distribute it freely under the terms of the license below.

Btw, this string is used in each of the about pages, so copyleft doesn't apply to it anyway, as it doesn't apply to all of the deps.

Reply:
This discussion is still open

Reply:
@tasn It is still copyleft without being strong https://en.wikipedia.org/wiki/Copyleft#Full_and_partial_copyleft
Translators usually leave the strings that are hard to translate, but overall this way of doing it works for other projects. I think explaining things without resorting to the full license is better, and that copyleft in itself is a very meaningful concept that shouldn't be hidden away in the license. Your version seems like it (possibly) even takes away from "use, modify and distribute freely". I'll go with that and "email" if that is what it takes.